src/uct/ib/rdmacm/rdmacm_def.h

```diff
@@ -25,6 +25,7 @@ typedef struct uct_rdmacm_ep      uct_rdmacm_ep_t;
 
 typedef struct uct_rdmacm_priv_data_hdr {
     uint8_t length;     /* length of the private data */
+    int8_t  status;
```

Comment:
looks like length and status are never used in the same time right? length is for requests, and status is for replies?
so let's separate to 2 different header typedefs

Reply:
For example it's used for assertion for all event types which sends the private data, i find the status of remote side is useful. Moreover, I see that rdmacm sends `UCT_RDMACM_UDP_PRIV_DATA_LEN` in any case if the data is present. I don't think that 1 byte is critical for address size. 2 header typedef will complicate the code.